docs/services/slam/cartographer/_index.md

```diff
@@ -260,7 +228,7 @@ Watch a map start to appear.
 | Name | Data Type | Inclusion | Description |
 | ---- | --------- | --------- | ----------- |
 | `data_dir` | string | **Required** | Path to [the directory](#slam-mapping-modes) used for saving output internal state in <file>/internal_state</file>. |
-| `sensors` | string[] | **Required** | Names of configured RPlidar devices providing data to the SLAM service. May not be empty. |
+| `sensors` | string | **Required** | Names of configured RPlidar devices providing data to the SLAM service. May not be empty. |
```

Comment:
it is an array of strings I think - otherwise it cannot be multiple
```suggestion
| `sensors` | string[] | **Required** | Names of configured RPlidar devices providing data to the SLAM service. May not be empty. |
```